### PR TITLE
LPS-120527 Expire article if article was manually expired.

### PR DESCRIPTION
**Pull request by marianoalvarosaiz**

### CODE REVIEW

**Comment by ruben-pulido** [comment]
ci:test:sf

**Comment by ruben-pulido** [comment]
ci:test:relevant

**Comment by marianoalvarosaiz** [comment]
Hi @ruben-pulido,
 do you have any updates about this PR?
Kind regards.

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalArticleStagedModelDataHandler.java [hunk 1]
```diff
@@ -1004,6 +1010,13 @@ else if (Validator.isNotNull(smallImagePath)) {
 				serviceContext.getAssetLinkEntryIds(),
 				serviceContext.getAssetPriority());
 
+			if (expired && !importedArticle.isExpired()) {
```
Comment (pavel-savinov) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalArticleStagedModelDataHandler.java:1013]: @marianoalvarosaiz @ruben-pulido podemos simplificarlo con 
`if (article.isExpired() && !importedArticle.isExpired()) {` ?

Comment (marianoalvarosaiz) [reply modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalArticleStagedModelDataHandler.java:1013]: Sí, sin problemas. Queréis que os reenvie la PR?

Comment (pavel-savinov) [reply modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/exportimport/data/handler/JournalArticleStagedModelDataHandler.java:1013]: Nope, tranqui, lo cambio y mando a Brian.

**Comment by pavel-savinov** [comment]
Just started reviewing :)

**Comment by pavel-savinov** [comment]
Sent here https://github.com/liferay-echo/liferay-portal/pull/2292
Gracias @marianoalvarosaiz

**Comment by pavel-savinov** [comment]
ci:close

**Pull request closed without merging**